src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -378,7 +390,7 @@ That will produce the following HTML:
 There are lots of ways to iterate over arrays in JavaScript with functions like `Array.map`, `Array.filter`, and `Array.reduce`.
 These all work in JSX!
 
-##### The `key` prop
+##### <span id="the-key-prop">The `key` prop</span>
```

Comment:
The markdown library didn't automatically generate an id for this heading so I manually added one for a link from the props module.